src/App.css

```diff
@@ -0,0 +1,151 @@
+
+
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+.App-header {
+  background-color: #282c34;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
+
+.App-link {
+  color: #61dafb;
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+#nav {
+  display:flex;
+  /* justify-content: center; */
+  background-color:#472d4f!important;
+  border-top: 2px solid #31e89f;
+  padding-bottom: 0;
+  padding-top: 0;
+  /* border-bottom: 1px solid #31c6e8; */
+}
+#dropdwn {
+  font-size: 30px;
+}
+#movieC {
+  padding: 0;
+  margin:5px;
+
+  
+}
+#display {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 20px;
+}
+#main {
+  background-color: #4c4c4c;
+}
+
+#searchBox {
+  border-color:#31e89f;
+  background-color: transparent;
+  color:lightgrey;
+}
+
+.jumbotron {
+  margin-bottom:0;
+  padding: 5.5rem 2rem;
+}
+.dropdown {
+  border-bottom: 1px solid #31e89f;
+  border-top: 1px solid #31e89f;
+  /* display:flex;
+  justify-content: center; */
+}
+.navbar-nav {
```

Comment:
pretty minor but keeping your CSS a little bit more organized would be nice. nav stuff should be together, for example.